recipes/bindsnet/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set name = "bindsnet" %}
+{% set version = "0.3.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/{{ name }}/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: 5d81f5502e357bef2a151461110c108a07292b96a16e383b179eba3023e418de
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ cdt('mesa-libgl-devel') }}  # [linux]
+    - {{ cdt('mesa-dri-drivers') }}  # [linux]
+    - {{ cdt('libselinux') }}  # [linux]
+    - {{ cdt('libxdamage') }}  # [linux]
+    - {{ cdt('libxxf86vm') }}  # [linux]
+    - {{ cdt('libxext') }}     # [linux]
+  host:
+    - pip
+    - python
+    - poetry
+    - xorg-libxfixes  # [linux]
+  run:
+    - cython
+    - gym >=0.10.4
+    - matplotlib-base
+    - numpy
+    - opencv
+    - pandas
+    - pytest >=3.4.0
```

Comment:
`pytest` shouldn't be a run requirement. It should only be needed for development.
It's not in https://github.com/BindsNET/bindsnet/blob/821fbc913f23acd1102ea0bce73defe54d5eee78/pyproject.toml#L12

Compare to that list `foolbox` is missing. As well as `scikit-build`.

Reply:
Thanks - I removed `pytest` and added `foolbox`.

While `scikit-build` is in their dependency list, it is actually not being used in BindsNet and I have thus not added it; if you feel strongly about it, I am happy to also add it.